modules/jupyter-widget/src/utils.js

```diff
@@ -89,24 +89,39 @@ function getTooltip(pickedInfo) {
 }
 
 function tabularize(json) {
+  // Turns a JSON object of picked info into HTML for a tooltip
   const dataTable = document.createElement('div');
   dataTable.className = 'dataTable';
 
+  // Creates rows of two columns for the tooltip
   for (const key in json) {
+    if (['undefined', 'position', 'index'].includes(key)) {
```

Comment:
- How could there ever be a key named `undefined`?
- Make the black list a constant so that you don't create a new array in each iteration. It's also faster to use `Set` instead of array.